runtime/doc/ui.txt

```diff
@@ -636,5 +637,29 @@ Only sent if `ext_wildmenu` option is set in |ui-options|
 ["wildmenu_hide"]
 	Hide the wildmenu.
 
+==============================================================================
+Floating window Events						      *ui-float*
+
+Only sent if `ext_float` option is set in |ui-options|.
+
+This extension allows the UI to draw floating windows (described at
+|nvim_open_float_win|) by receieving updates on separate grids. Using this
+extension implicitly activates `ext_newgrid` extension. Floats will be drawn
```

Comment:
IMO it would be preferable to "require" `ext_newgrid` (error if missing), rather than implicitly enabling stuff. Otherwise the list of enabled extensions does not match the explicit list sent by the client.

Reply:
lets integrate the docs with the "multigrid" selection. The floats feature can be relegated to a debug flag only, as we discussed on gitter.